includes/class-edd-session.php

```diff
@@ -300,13 +304,56 @@ public function use_cart_cookie() {
 	}
 
 	/**
+	 * Determines if we should start sessions
+	 *
+	 * @since  2.5
+	 * @return bool
+	 */
+	public function should_start_session() {
+
+		$start_session = true;
+
+		if( ! empty( $_SERVER[ 'REQUEST_URI' ] ) ) {
+
+			$blacklist = apply_filters( 'edd_session_start_uri_blacklist', array(
+				'feed',
+				'feed',
+				'feed/rss',
+				'feed/rss2',
+				'feed/rdf',
+				'feed/atom',
+				'comments/feed/'
+			) );
+
+			$uri = ltrim( $_SERVER[ 'REQUEST_URI' ], '/' );
+			$uri = untrailingslashit( $uri );
+
+			if( in_array( $uri, $blacklist ) ) {
```

Comment:
This breaks for domains like `http://local.dev/edd/feed/` where you have WordPress in a subdirectory like multisite or even just a sub directory base install since it's looking for the request URI to be just `feed` not `edd/feed` which is the result of the `ltrim` and `untrailingslashit`